doc/repoquery.rst

```diff
@@ -75,6 +75,9 @@ Together with ``<pkg-spec>``, control what packages are displayed in the output.
 ``--whatrequires <capability>``
     Limit the resulting set only to packages that require ``<capability>``.
 
+``--alldeps``
+    Expand the resulting set of --whatrequires to packages that also require files that ``<capability>`` provides.
```

Comment:
Sorry, I just can't understand it. I didn't get what it should do. But maybe it's just me...